src/sentry/models/groupinbox.py

```diff
@@ -116,7 +129,11 @@ def remove_group_from_inbox(group: Group, action=None, user=None, referrer=None)
         pass
 
 
-def bulk_remove_groups_from_inbox(groups, action=None, user=None, referrer=None):
+def bulk_remove_groups_from_inbox(
+    groups: BaseQuerySet[Group, Group],
+    action: GroupInboxRemoveAction | None = None,
+    user: User | RpcUser | Team | AnonymousUser | None = None,
```

Comment:
Adding these types because `record_group_history` expects them.
https://github.com/getsentry/sentry/blob/5b7333552e7a566f889da2e4e954d84ee8d39861/src/sentry/models/grouphistory.py#L286-L317